.github/workflows/ci.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - master
+      - "greenkeeper/*"
```

Comment:
Github Actions won't match a Greenkeeper branch that has an extra slash in its name (i.e. scoped packages). This `greenkeeper/*` should be turned into `greenkeeper/**`.

Reply:
yeah I just realized and fixed it, cheers

Reply:
Resolved via f1c02cb.